### ISSUE TITLE
Query results differ

### ISSUE THREAD

**Issue by ddu2019** [LisaHJung/Part-4-Running-Aggregations-with-Elasticsearch-and-Kibana#4]
First of all, thank you for this excellent beginner's crash course.

On Part 4 of the tutorial, I did not get the same results as desribed in this repo or in the video.

However, if I replace the 0 with a 1 in the STEP 3 query, the results were again matching.

**STEP 3: Remove the negative values from the field "UnitPrice".**

_ORIGINAL_

```
POST ecommerce_data/_delete_by_query
{
  "query": {
    "range": {
      "UnitPrice": {
        "lte": 0
      }
    }
  }
}
```

_MODIFIED_

```
POST ecommerce_data/_delete_by_query
{
  "query": {
    "range": {
      "UnitPrice": {
        "lte": 1
      }
    }
  }
}
```

Keep up the good work  :+1: !!!

**Comment by Moandh81** [comment]
> 

Thank you so much for mentioning this.
I have now the same values as in the repo

**Comment by nickchomey** [comment]
Yeah, I just discovered the same thing - assumed she had used 1 since her min value was 1.01 rather than the 0.001 that I was receiving.
